src/recv.c

```diff
@@ -32,10 +32,41 @@
 #include "internal.h"
 #include "iasync.h"
 #include "sync.h"
+#include "binary.h"
 
 #include <string.h>
 #include <stdlib.h>
 
+char * 
+mpd_recv_binary(struct mpd_connection *connection, const unsigned binary) {
+	char * data = malloc(binary);
+	assert(data);
```

Comment:
`assert()` is the wrong tool to check for runtime errors.

Reply:
changed